content/cloud/pricing-and-billing.md

```diff
@@ -94,10 +94,10 @@ No. Spin remains 100% open source and we remain highly committed to both the Spi
 
 For an up-to-date list of Fermyon’s data subprocessors please visit https://www.fermyon.com/sub-processors.
 
-**Can I purchase a Service Level Agreement?**
+**Can I purchase a Support Service Level Agreement?**
 
 As part of Fermyon Cloud's Starter Plan you have access to our vibrant discord community and as part of the Growth Plan you have access to next business day support through [support@fermyon.com](mailto:support@fermyon.com)
```

Comment:
Why "vibrant"? It sounds fun, but maybe I'm not looking for fun if I need support :-)